index.md

```diff
@@ -138,17 +138,19 @@ There are more native clients in development, Nostros[^nostros] and Nozzle[^nozz
 
 Web clients:
 
-- [iris.to](https://iris.to/) - Clean interface & rich in features
 - [primal.net](https://primal.net/) - Explore your tribe, network, and global trends
 - [snort.social](https://snort.social/) - Simple interface with automatic image-upload
+- [noStrudel](https://nostrudel.ninja/) - Supports many NIPs inc communities, streams, blogs and more
 - [coracle.social](https://coracle.social/) - Search, filters, and micro-apps
 - [yosup.app](https://yosup.app/) - Mobile-friendly and twitter-like
+- [iris.to](https://iris.to/) - Clean interface & rich in features
 - [nostrgram.co](https://nostrgram.co/) - Focus on images and media, supports multiple layout styles
 
-On Android, you can use the [Kiwi Browser](https://kiwibrowser.com/) which
-allows you to install Alby or nos2x, which in turn allows you to use any web
-client. [Yosup](https://yosup.app/) and [Hamstr](https://hamstr.to/) have good
-mobile experiences, for example.
+On Android, you can use:
+
+- [Plebstr](https://play.google.com/store/apps/details?id=com.plebstr.client) - Good short message client similar to Twitter.
```

Comment:
Plebstr should be where Amethyst is (Android clients section above)

Reply:
Agreed. if you merge this I'll fix it with the next pull request.

Reply:
Merged it & I've just made the change on `master`.

Thanks again for your contribution! 🙏💜

Reply:
great. I'll sync up.